pages.ar/linux/dockerd.md

```diff
@@ -0,0 +1,24 @@
+# dockerd
+
+> هي عملية مستمرة تعمل في الخلفية تبدأها لتتحكم في حاويات الدوكر.
+> لمزيد من التفاصيل: <https://docs.docker.com/engine/reference/commandline/dockerd/>.
+
+- قم بتشغيل دوكر في الخلفية:
+
+`dockerd`
+
+- قم بتشغيل دوكر في الخلفية واجعله يستمع علي منفذ معين (UNIX and TCP):
+
+`dockerd --host unix://{{path/to/tmp.sock}} --host tcp://{{ip}}`
```

Comment:
Could you please also translate the arguments if possible?

Reply:
I added the translation of the args.